arkouda/numeric.py

```diff
@@ -428,26 +426,34 @@ def cos(pda: pdarray) -> pdarray:
     return create_pdarray(repMsg)
 
 
-def _hash_helper(a: hashable):
-    from arkouda import SegArray as Segarray_
+def _hash_helper(a: Union[pdarray, Strings, "SegArray", Categorical]):  # type: ignore
```

Comment:
to be perfectly honest, i tried to get the two in line but this is the only configuration i could get that avoided typing error or mypy failures. I can def play around with this some more

Reply:
I am just wondering it the `# type: ignore` is why it is working or not. I think it may warrant some discussion, just to make sure we know what is happening.